fix: links with same path but different search params not prefetched (#9189)
withastro/astro: .changeset/brown-jars-lick.md

---
'astro': patch
---

> fix: links with same path but different search params not prefetched

Reviewer: ```suggestion
Enables prefetching links with the same current path but different search params
```
